### DISCUSSION TITLE
Issues with change in AWSCLI versions

### DISCUSSION THREAD

**Discussion by oldgal43** [aws/aws-cli#7140] [Q&A] [answered]
If I move to AWSCLI version 2 from version 1, will there be any changes in the commands?  Will the same buckets be available in both versions?  I

I am getting "An error occurred (AccessDenied) when calling the ListObjects operation: Access Denied"

**Comment by aBurmeseDev** [comment, marked as answer]
Hi @oldgal43, thanks for reaching out! 

There are new changes in v2 that might require you to update scripts and commands and here's [v2 doc](https://docs.aws.amazon.com/cli/latest/userguide/cliv2-migration-changes.html#cliv2-migration-changes-breaking) for new features and breaking changes for your reference.

In your case, however, you might have to attach IAM policy that allows the `ListBucket` action on the bucket itself and the `GetObject` action on all of the bucket's objects to your IAM user/role. Please check out [this stackOverflow post](https://stackoverflow.com/a/62940788) for code example.
